src/compiler/diagnosticMessages.json

```
@@ -7624,6 +7624,10 @@
         "category": "Message",
         "code": 95181
     },
+    "Only variables declared exactly once can be inlined.": {

```

A variable is _one_ possible declaration of a _symbol_ with multiple declarations, but a symbol having multiple declarations isn’t how users think about values/types/namespaces overlapping.

```suggestion
    "Variables that share a name with a type or namespace in the same scope cannot be inlined.": {
```